lib/utils/action-hook-delay.js

```diff
@@ -126,7 +126,12 @@ class ActionHookDelayProcessor extends Emitter {
     try {
       this._taskInProgress = makeTask(this.logger, t[0]);
       this._taskInProgress.disableTracing = true;
-      this._taskInProgress.exec(this.cs, {ep: this.ep});
+      this._taskInProgress.exec(this.cs, {ep: this.ep}).catch((err) => {
+        this.logger.info(err, 'ActionHookDelayProcessor#_onNoResponseTimer: error playing file');
```

Comment:
logging err object causing the entire trace in the log file, can we log err.message>